packages/fractor-rule-generator/src/Console/Command/GenerateRuleCommand.php

```diff
@@ -0,0 +1,242 @@
+        private readonly FileGenerator $fileGenerator,
+        private readonly OutputInterface $outputStyle,
+        private readonly ConfigFilesystem $configFilesystem,
+        private readonly FileInfoFactory $fileInfoFactory
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        /** @var Typo3Version $typo3Version */
+        $typo3Version = $helper->ask($input, $output, $this->askForTypo3Version());
+        $changelogUrl = $helper->ask($input, $output, $this->askForChangelogUrl());
+        $name = $helper->ask($input, $output, $this->askForName());
+        $description = $helper->ask($input, $output, $this->askForDescription());
+        $type = $helper->ask($input, $output, $this->askForType());
+
+        $recipe = new Typo3FractorRecipe(
+            $typo3Version,
+            $changelogUrl,
+            $name,
+            $description,
+            Typo3FractorTypeFactory::fromString($type)
+        );
+
+        $templateFileInfos = $this->templateFinder->find($recipe->getFractorFixtureFileExtension());
+
+        $templateVariables = [
+            '__MajorPrefixed__' => $recipe->getMajorVersionPrefixed(),
+            '__Major__' => $recipe->getMajorVersion(),
+            '__MinorPrefixed__' => $recipe->getMinorVersionPrefixed(),
+            '__Type__' => $recipe->getFractorTypeFolderName(),
+            '__FixtureFileExtension__' => $recipe->getFractorFixtureFileExtension(),
+            '__Name__' => $recipe->getFractorName(),
+            '__Test_Directory__' => $recipe->getTestDirectory(),
+            '__Changelog_Annotation__' => $recipe->getChangelogAnnotation(),
+            '__Description__' => addslashes($recipe->getDescription()),
+            '__Use__' => $recipe->getUseImports(),
+            '__Traits__' => $recipe->getTraits(),
+            '__ExtendsImplements__' => $recipe->getExtendsImplements(),
+            '__Base_Fractor_Body_Template__' => $recipe->getFractorBodyTemplate(),
+        ];
+
+        $targetDirectory = __DIR__ . '/../../../../typo3-fractor';
+
+        $generatedFilePaths = $this->fileGenerator->generateFiles(
+            $templateFileInfos,
+            $templateVariables,
+            $targetDirectory
+        );
+
+        $this->configFilesystem->addRuleToConfigurationFile(
+            $recipe->getSet(),
+            $templateVariables,
+            self::FRACTOR_FQN_NAME_PATTERN
+        );
+
+        $testCaseDirectoryPath = $this->resolveTestCaseDirectoryPath($generatedFilePaths);
+        $this->printSuccess($recipe->getFractorName(), $generatedFilePaths, $testCaseDirectoryPath);
+
+        return Command::SUCCESS;
+    }
+
+    private function askForTypo3Version(): Question
+    {
+        $whatTypo3Version = new Question('TYPO3-Version (i.e. 12.0): ');
```

Comment:
e.g. ("exempli gratia = for example"), not i.e. ("id est = that is")

There's various other places with that error as well.